src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
@bluong wrote:
> Whenever I see `d3.select`, I personally get scared that it'll look throughout the entire page for the selector, which can lead to bugs sometimes

In this case though, we're selecting an existing element, not using a selector string. Previous bugs caused by `d3.select()` were due to passing in a selector string.